exercises/trinary/example.js

```diff
@@ -14,5 +14,7 @@ Trinary.prototype.toDecimal = function () {
 };
 
 Trinary.prototype.accumulator = function (decimal, digit, index) {
-  return decimal += digit * Math.pow(BASE, index);
```

Comment:
The error ESLint throws is

```
/home/txingo/repositories/exercism-repositories/javascript/exercises/trinary/example.js
  17:3   error  Return statement should not contain assignment  no-return-assign
  17:10  error  Assignment to function parameter 'decimal'      no-param-reassign
```

It might be due to `+=`. And it makes sense. What's the point of assignning a value and then return it? Why not return it directly?

```
return decimal + digit ...
```

I tested, and just removing the `=` sign makes ESLint happy, and tests still pass.